SingularityUI/app/components/taskSearch/DisplayResults.cjsx

```diff
@@ -0,0 +1,242 @@
+                paramValue = @props.lastTaskStatus
+                onClick = @props.clearLastTaskStatus
+                /></div>)
+            key++
+        if @collection.params.startedAfter
+            params.push(<div key={key}> <QueryParam
+                paramName = "Started After"
+                paramValue = {@props.startedAfter.format window.config.timestampWithSecondsFormat}
+                onClick = @props.clearStartedAfter
+                /></div>)
+            key++
+        if @collection.params.startedBefore
+            params.push(<div key={key}> <QueryParam
+                paramName = "Started Before"
+                paramValue = {@props.startedBefore.format window.config.timestampWithSecondsFormat}
+                onClick = @props.clearStartedBefore
+                /></div>)
+            key++
+        if @collection.params.sortDirection
+            params.push(<div key={key}> <QueryParam
+                paramName = "Sort Direction"
+                paramValue = @props.sortDirection
+                onClick = @props.clearSortDirection
+                /></div>)
+            key++
+        return params
+
+    updatePageNumber: (event) ->
+        @setState({
+            pageNumberEntered: event.target.value
+        })
+
+    handlePageJump: (event) ->
+        event.preventDefault()
+        @props.setPageNumber(@state.pageNumberEntered)
+        @setState({
+            pageNumberEntered: ''
+        })
+
+    updateCount: (event) ->
+        @props.updateCount(event.target.value)
+
+    # using className="previous" for the next button is necessary to align
+    # it to the left side of the page. This is built into bootstrap
+    renderPageToggles: ->
+        <div className="container-fluid">
+            <nav>
+                <ul className="pager line">
+                    <li className={@collection.params.page == 1 and "previous disabled" or "previous"} onClick={@props.decreasePageNumber}><a href="#">Previous</a></li>
+                    <li className="previous disabled"><a href="#">Page {@collection.params.page}</a></li>
+                    <li className="previous" onClick={@props.increasePageNumber}><a href="#">Next</a></li>
+                    <form role="form" onSubmit={@handlePageJump} className='form-inline text-left'>
+                        <div className='form-group'>
+                            <label htmlFor="pageNumber" className="sr-only">Jump To Page:</label>
+                            <FormField 
+                                value = @state.pageNumberEntered 
+                                inputType = 'number'
+                                id = 'pageNumber'
+                                title = "Jump to Page"
+                                updateFn = @updatePageNumber />
+                        </div>
+                        <button type="submit" className="btn btn-default">Jump!</button>
+                        &nbsp;&nbsp;
```

Comment:
Usually it's preferable to add margin or padding with CSS rather than adding non-breaking spaces in the markup, since it makes restyling easier.